runtime/oti/j9nonbuilder.h

```diff
@@ -3224,6 +3225,9 @@ typedef struct J9Class {
 #define J9CLASS_PREPADDING_SIZE(clazz) 0
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 
+/* For the following, J9_ARE_ANY_BITS_SET fails on zOS, currently under investigation */
```

Comment:
can you add the issue number to this comment

Reply:
Done